development/README.md

```diff
@@ -1,11 +1,33 @@
 # Development
 
-## Bug
-We define a bug as any experience that breaks the _intended_ functionality of a user experience in [Tilt](https://docs.tilt.dev/), including [Tilt Cloud](https://cloud.tilt.dev/). In particular, we may have shipped a feature that failed to meet a certain user's expectation, but that may not necessarily qualify as a bug. In this case, we may should nonetheless track the user problem as a non-bug to improve Tilt.
+## Bugs
+
+### Definition
+We define a bug as any experience that breaks the _intended_ functionality of a user experience in [Tilt](https://docs.tilt.dev/), including [Tilt Cloud](https://cloud.tilt.dev/). In particular, we may have shipped a feature that failed to meet a certain user's expectation, but that may not necessarily qualify as a bug. In this case, we should nonetheless track the user problem as a non-bug to improve Tilt.
 
 We should express our intentions through [documentating Tilt functionality](https://docs.tilt.dev/) exhaustively, to better support users and manage their expectations.
 
 Non-user facing problems are not classified as bugs.
 
-## Tracking work
-See the higher-level [Product Development workflow](../product-development/README.md#workflow) workflow focused on prioritizing and solving business problems. This section is focused on tracking more granular development work on a daily and weekly basis.
+### All bugs
+All known open bugs should be listed in the [Bugs Clubhouse space](https://app.clubhouse.io/windmill/stories/space/4729/bugs). Some bugs are also listed in [GitHub issues](https://github.com/windmilleng/tilt/labels/bug). But these should be a subset of the entire set of open bugs in the Clubhouse space.
+
+### Fix a bug at anytime
+We believe in shipping a quality experience for both Tilt and Tilt Cloud. So Tilters are encouraged to use their own judgment in picking up a bug to fix at anytime, if they think it's the best use of their time at that moment, ahead of other priorities. Nobody should ever need permission to fix a bug.
```

Comment:
I feel like there's an implicit assumption here that a bug can be fixed quickly. It might be worth saying a few words about bugs that can't be fixed quickly - e.g., the kind of bug that would take a multi-week refactor

Reply:
Added this: https://github.com/windmilleng/company/pull/21/commits/50e840af0ab8756afdf538ab18792e3cea32d609